dev-docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Official Solr Dockerfile Management
+
+In general most interactions with this repository should be done via the Solr Release Wizard, not manually.
+
+## How an Official Solr Dockerfile is released
+
+1. In the Solr Release Wizard, an **official** Dockerfile will be created as a part of the release candidate.
+   The release candidate is tested and voted on, including the Dockerfile.
```

Comment:
Nitpick - The dockerfile is not technically voted upon? Else we'd have needed a respin of 9.0 to fix review requests on official Dockerfile... In 99 out of 100 times we'll use the Dockerfile from the RC, but if we need to fix a bug in that Dockerfile, e.g. to mitigate some vulnerability in the underlying JDK or Linux, then we should be able to push such a Dockerfile-only fix without another vote.. Don't know if it is worth a note in this doc?

Reply:
Fair enough, that's a good point. I think my point is made without this sentence. Basically we have a blanket rule that we will not change already released Dockerfiles, but we can make exceptions when we need to (like we did for 9.0.0).

Reply:
Fixed, added that the image is explicitly not "voted on", and why that is (in case that information is lost to future maintainers).

Also added a section on how changes should be made if the official-images team request them. (And called out that all relevant changes need to be ported to `apache/solr`)